### DISCUSSION TITLE
Can this be fixed?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#84728] [New to GitHub]
### Select Topic Area

Bug

### Body

→ ~ cd github-changelog
→ ~/github-changelog|main git log main
showing all changes successfully "(run)"

**Comment by ghostinhershell** [comment]
Welcome to the GitHub Community, @homemakeremai, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by faridnec** [comment]
Hey, can you provide more context about the issue? Here I suppose you are navigating to the github-changelog directory and then attempting to view the git log for the main branch. The output "showing all changes successfully" indicates that the git log for the main branch was displayed successfully.

There doesn't seem to be an error or anything to be fixed

**Reply by ghost** [reply]
How do I display my new unlocked achievements?

**Reply by faridnec** [reply]
If you unlocked achievements it should have displayed under your profile automatically. Sometimes there is case when it bugs and does not display which previously happened to me. In that case I solved it by go to this session on your browser
```markdown
https://github.com/<your-github-username>?tab=achievements
```

this will display your unlocked achievements. Click one of those achievements and you will see an option to hide and unhide it. If your achievement is not present under your main profile just try to hide and unhide it again and in my case it worked and it appears there. Hope this helps!
![Uploading gitty.png…]()

**Comment by ghost** [comment]
I'm all set then, and it will be alright.
